test/apps/react-oie/src/App.js

```diff
@@ -25,9 +26,33 @@ const formMetaMapper = (nextStep) => {
 };
 
 export default function App() {
+  const history = useHistory();
+  const flowMethodRef = useRef('');
   const [transaction, setTransaction] = useState(null);
-  const [flowMethod, setFlowMethod] = useState('');
   const [inputValues, setInputValues] = useState({});
+  const [authState, setAuthState] = useState(null);
+
+  useEffect(() => {
+    const parseFromUrl = async () => {
+      try {
+        await oktaAuth.idx.handleInteractionCodeRedirect(window.location.href);
+        history.push('/');
+      } catch (err) {
+        console.log(err);
+      }
+    };
+
+    const updateAuthState = authState => {
+      setAuthState(authState)
+    };
+
+    oktaAuth.authStateManager.subscribe(updateAuthState);
+    oktaAuth.start();
+
+    if(oktaAuth.isLoginRedirect()) {
+      parseFromUrl();
+    }
+  }, []);
```

Comment:
your dependency array should consist of `[history, setAuthState]`, since they are used in the Effect

Reply:
looks like a linter should be added